css/index.css

```diff
@@ -104,14 +196,45 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
+    background: black;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+#one {
+    background: teal;
+}
+#two {
+    background: gold;
+}
+#three {
+    background: cadetblue;
+}
+#four {
+    background: coral;
+}
+#five {
+    background: crimson;
+}
+#six {
+    background: forestgreen;
+}
+#seven {
+    background: darkorchid;
+}
+#eight {
+    background: hotpink;
+}
+#nine {
+    background: indigo
+}
+#ten {
+    background: dodgerblue;
+}
+
```

Comment:
I would use classes here instead of ID's.
Remember you only want to use ID's for things you may only use once.